05_stringer/sirigithub/main_test.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+	main()
+	expected := "127.0.0.1"
+	actual := buf.String()
+	assert.Equal(t, expected, actual, "Unexpected output in main() actual %v but expected %v", actual, expected)
+}
+func TestIPAddrStringer(t *testing.T) {
+
```

Comment:
Nit: Unnecessary blank line.
No need for blank lines at beginning or end of block.
Use blank lines between type, method and function definitions and within a block for logical separation.